ykrt/try_passes.py

```diff
@@ -130,19 +128,27 @@ def attempt2_inner(logf, ok_passes, try_passes, flagType):
         progress_meter()
         return
     else:
-        subset_len = len(try_passes) // 2
-        subset1 = try_passes[:subset_len] 
-        subset2 = try_passes[subset_len:]
+        subset_len = len(try_passes)
+        subset1 = try_passes[:subset_len // 2] 
+        subset2 = try_passes[subset_len // 2:]
         # XXX: We assume that whatever subsets we accept from this splitting
         # work in combination, but that may not be true. Not sure what we'd do
         # if they didn't work in combination though... restart the whole
         # search?
-        p1 = Process(target=attempt2_inner, args=(logf, ok_passes, subset1, flagType,))
```

Comment:
Is the XXX comment above still true? Based on our chat last week, I believe so?

Reply:
Yes, we still make some combinations.